eq3bt/bleakconnection.py

```diff
@@ -79,10 +83,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self._loop.run_until_complete(self._conn.disconnect())
             self._conn = None
 
-    async def on_notification(self, handle, data):
+    async def on_notification(self, characteristic, data):
         """Handle Callback from a Bluetooth (GATT) request."""
         # The notification handles are off-by-one compared to gattlib and bluepy
-        handle = handle + 1
+        handle = characteristic.handle + 1
```

Comment:
Considering this is a breaking change in the upstream lib, the version pinning for bleak should probably be updated (alongside the lock file created by `poetry update` afterwards).

Alternatively, we could simply do the following to avoid pinning (preferred way, I think):
```
try:
    handle = characteristic.handle + 1
except AttributeError:  # TBD: change to correct exception type
    handle = handle + 1
```

Reply:
Hello, 
Interested in this fix as it affects eq3bt integration in HASS latest version.
https://www.home-assistant.io/blog/2022/07/06/release-20227/#breaking-changes

Reply:
@kazcrn-omnisend In the meanwhile, here is a "non-compliant" custom component: https://github.com/dbuezas/eq3btsmart